README.md

```diff
@@ -432,8 +431,7 @@ The SD card is now ready to be booted by the DevTerm! Good luck!
 
 ## Next Steps
 
-You will want to set up Wi-Fi on first boot. You can do so by
```

Comment:
could you also test if wiringpi and the printer works?
also, while I'm not sure if it works, you can test suspend without the mainboard mod like this:
```
1. sudo systemctl suspend
2. observe that it goes to sleep
3. plug in power
4. observe whether it resumes
```

Reply:
I don't have the printer to try it sorry.

Reply:
And fan works 👍

Reply:
ooh then it means we actually have a way to wake up the core without the mainboard hack. this is good.
no worry about the printer.